sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -396,11 +396,20 @@ def test_binomial():
     z = Symbol('z', zero=True)
     nz = Symbol('z', zero=False)
 
-    for i in range(-1, 2):
-        for j in range(-1, 2):
-            assert binomial(i, i - j) == binomial(i, j)
-            assert i == j == 0 or binomial(i, j
```

Comment:
This is the condition I was talking about @smichr

Reply:
I replaced it with a generalised test. and that turned out to evaluate to None. IIRC you added these tests remember ?

Reply:
```
>>> i, j = symbols('i, j',integer=True)
>>> binomial(i, j).equals(binomial(i - 1, j - 1) + binomial(i - 1, j))
True
```
Isn't this wrong? We don't know if `i=j==0` and if it does, this doesn't hold.

Reply:
> Isn't this wrong? We don't know if `i=j==0` and if it does, this doesn't hold.

Yeah it's wrong! I added it to the TODO @ OP 😄